packages/subgraph/networks.json

```diff
@@ -12,5 +12,33 @@
             "address": "0xc9E426B468d334AF6208ef9b3bA5f599d1417c6e",
             "startBlock": 28680520
         }
+    },
+    "scroll-sepolia": {
```

Comment:
Scroll Sepolia has a subgraph?

Reply:
Not really but the new autogen script that generates this file doesn't make distinctions. If a network has a Carrot deployment it ends up here, even if TheGraph doesn't support the network. I think it's fine as the ultimate source of truth on where subgraphs are deployed are the package.json scripts